apache/superset
# chore: Select component refactoring - TimeSeriesColumnControl - Iteration 5

### SUMMARY
It replaces the react-select Select component with the Antdesign one in the TimeSeriesColumnControl component.

### BEFORE

https://user-images.githubusercontent.com/60598000/130768485-f694c7ec-b6e3-43d0-b178-93a8b7326be8.mp4

### AFTER

https://user-images.githubusercontent.com/60598000/130768576-9d61f25f-c677-490b-b16b-ce2130f441df.mp4

### TESTING INSTRUCTIONS
1. Open a Time-series Table Chart in Explore
2. Add time columns and observe the dropdown behavior

### ADDITIONAL INFORMATION
